docker/dockerfiles/be/upload_coredump.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/bash
```

Comment:
this script is not packed into artifacts and be/cn image, missing the change in corresponding dockerfile?